.ci/Jenkinsfile-compile

```diff
@@ -99,7 +99,8 @@ pipeline {
                      "px4_fmu-v6x_default",
                      "px4_io-v2_default",
                      "spracing_h7extreme_default",
-                     "uvify_core_default"
+                     "uvify_core_default",
+                     "yuneec_mantis-edu_default"
```

Comment:
We'll have to rename `yuneec` to `atl` everywhere.

Reply:
Done.